build.gradle

```diff
@@ -9,7 +9,14 @@ repositories {
 	}
     maven { url 'https://jitpack.io' }
     maven {
-        url 'https://maven.pkg.github.com/RHEcosystemAppEng/exhort-java-api'
+        url 'https://maven.pkg.github.com/trustification/exhort-java-api'
```

Comment:
this should be part only of the ci workflow

Reply:
Sorry, I meant the use of the GitHub environment variables, not this line in specific.

Reply:
Hi, I see they're well configured in CI actions such as:
```
        -Pgpr.username=${{ github.actor }}
        -Pgpr.token=${{ secrets.GITHUB_TOKEN }}
```
Do you want me to just remove the credentials part? Maybe add a similar section about registry access configuration in `README`?

Reply:
Yes, please, remove them from the build.gradle file.
There's no need to add it to the README as it is already documented in the [`exhort-java-api` docs](https://github.com/trustification/exhort-java-api?tab=readme-ov-file#usage)

Reply:
update as per review comment. Thanks.